docs/UserGuide.md

```diff
@@ -309,6 +309,38 @@ Skills: [Accounting][Equities][Cryptocurrency]
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
+---
+### Sorting applicants : `sort`
+
+Allows you to sort the applicant list temporarily using the parameter identified by the field. The order of sorting is based on the input order.
```

Comment:
A bit too technical for UG imo, possible to replace the word `parameter` and hide the details

```suggestion
Allows you to sort the list of applicants temporarily by name or by job. You can sort based on ascending or descending order.
```